### DISCUSSION TITLE
Welcome to skarf Discussions!

### DISCUSSION THREAD

**Discussion by woooferz** [woooferz/skarf#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xstar97** [comment]
@woooferz where's the docker image located on dockerhub? i want to add it to truecharts repo for truenas scale

**Comment by woooferz** [comment]
it is currently not on docker hub but you should be able to pull the image with `docker pull ghcr.io/woooferz/skarf:sha-d7474a7`, should i add it to docker hub?

**Reply by xstar97** [reply]
@woooferz if you can

**Reply by woooferz** [reply]
ok ive started the new github actions, hopefully it works with docker.io now

**Reply by woooferz** [reply]
ok it works with `docker pull wooferz/skarf:master` soon it will work with the dev branch as well

**Reply by xstar97** [reply]
@woooferz TY.
